src/openloco/windows/construction/Construction.h

```diff
@@ -0,0 +1,523 @@
+#pragma once
+
+#include "../../companymgr.h"
+#include "../../date.h"
+#include "../../graphics/colours.h"
+#include "../../graphics/image_ids.h"
+#include "../../input.h"
+#include "../../interop/interop.hpp"
```

Comment:
The vast majority of these includes are not required. Prune them so that only the required are left.